example/example.client.ts

```diff
@@ -426,28 +440,37 @@ export class ExpressZodAPIClient {
     request: K,
     params: Input[K],
   ): Promise<Response[K]> {
-    const [method, path] = request.split(/ (.+)/, 2) as [Method, Path];
-    return this.implementation(
-      method,
-      Object.keys(params).reduce(
-        (acc, key) =>
-          acc.replace(`:${key}`, (params as Record<string, any>)[key]),
-        path,
-      ),
-      Object.keys(params).reduce(
-        (acc, key) =>
-          Object.assign(
-            acc,
-            !path.includes(`:${key}`) && {
-              [key]: (params as Record<string, any>)[key],
-            },
-          ),
-        {},
-      ),
-    );
+    const [method, path] = parseRequest(request);
+    return this.implementation(method, ...substitute(path, params));
   }
 }
 
+export const subscribe = <
+  K extends Extract<Request, `get ${string}`>,
+  R extends Extract<PositiveResponse[K], { event: string }>,
+>(
+  request: K,
+  params: Input[K],
+) => {
+  const [path, rest] = substitute(parseRequest(request)[1], params);
+  const source = new EventSource(
```

Comment:
it may require `withCredentials: true`

Reply:
perhaps can do it automatically by comparing `window.location.origin` to `new URL().origin`

Reply:
no, this is for cookies